src/app/pages/explore/search/search.module.ts

```diff
@@ -8,14 +8,16 @@ import {ComponentModule} from 'src/app/components/component.module';
 import {SearchPage} from './search.page';
 import {SearchResultComponent} from './search-result/search-result.component';
 import {SearchCardComponent} from './search-result/search-card/search-card.component';
+import {NgZorroAntdModule} from "ng-zorro-antd";
 
 @NgModule({
     imports: [
         CommonModule,
         FormsModule,
         IonicModule,
         SearchPageRoutingModule,
-        ComponentModule
+        ComponentModule,
+        NgZorroAntdModule
```

Comment:
chỗ này không phải add NgZorroAntdModule, trong toàn app, chỉ add 1 lần duy nhất tại appModule. Việc add 1 module quá to vào sẽ làm app trở nên nặng.
ở những module con, khi cần cần sử dụng component nào, thì mới add Module  chứa component đó, 
vd: nz-pagination thuộc module NzPaginationModule, nên chỉ cần thêm NzPaginationModule là được